index.html

```diff
@@ -39,21 +40,17 @@
         <header>
           <nav class="nav-bar wrap-items">
             <a href="#">Nelson Ongiti</a>
+            <div class="openMenu"><i class="fa fa-bars"></i></div>
             <ul class="nav-links">
-              <li class="portfolio">
-                <a href="#works" class="menu">Porfolio</a>
-              </li>
-              <li class="about">
-                <a href="#about" class="menu">About</a>
-              </li>
-              <li class="contact">
-                <a href="#contact" class="menu">Contact</a>
-              </li>
+              <a  class="menu" href="#works">Portoflio</a>
+              <a  class="menu" href="#about">About</a>
+              <a class="menu" href="#contact">Contact</a>
+              <div class="closeMenu"><i class="fa fa-times"></i></div>
```

Comment:
- [ ] [REQUIRED] Please, bring back the `<li>` to wrap the `<a>` tags for semantic reasons. Since the `<ul>` can't have any direct children other than the `<li>` elemnets.

Reply:
## changes Made
- included the Li tags in the menu links